articles/TOC.yml

```diff
@@ -10,7 +10,7 @@
     href: overview/why-learn-quantum-computing.md
   - name: What are Q# and the QDK?
     href: overview/what-is-qsharp.md
-  - name: How can I learn more?
+  - name: How to learn quantum computing?
```

Comment:
I thought Carolyn pointed out that it shouldn't have a "?" since it's not a question

Reply:
Same as above, my bad